packages/admin-next/dashboard/src/components/grid/data-grid/data-grid-root/data-grid-root.tsx

```diff
@@ -0,0 +1,627 @@
+import { clx } from "@medusajs/ui"
+import {
+  ColumnDef,
+  Row,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table"
+import { useVirtualizer } from "@tanstack/react-virtual"
+import {
+  MouseEvent as ReactMouseEvent,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from "react"
+import { FieldValues, Path, UseFormReturn } from "react-hook-form"
+
+import {
+  Command,
+  useCommandHistory,
+} from "../../../../hooks/use-command-history"
+
+type FieldCoordinates = {
+  column: number
+  row: number
+}
+
+export interface DataGridRootProps<
+  TData,
+  TFieldValues extends FieldValues = any,
```

Comment:
nit: 
```suggestion
  TFieldValues extends FieldValues = FieldValues,
```

Other places in this file as well if you agree.